recipes/install.rb

```diff
@@ -890,3 +890,13 @@
   mode "770"
   action :create
 end
+
+# Add logo for opensearch-dashboards
```

Comment:
do we need this? does the rebranding work?

Reply:
removed